packages/account-v2/src/modules/IDVService/IDVService.tsx

```diff
@@ -0,0 +1,59 @@
+// [TODO]: Remove this eslint disable rule once form submission is implemented
+import React, { Fragment } from 'react';
+import { Formik } from 'formik';
+import { useKycAuthStatus } from '@deriv/api-v2';
+import { Button, Divider, Text } from '@deriv-com/ui';
+import { PersonalDetailsFormWithExample } from '../../containers';
+import { getIDVFormValidationSchema } from '../../utils/idvFormUtils';
+import { getNameDOBValidationSchema } from '../../utils/personal-details-utils';
+import { IDVForm } from '../IDVForm';
+
+type TIDVServiceProps = {
+    countryCode: string;
+    supportedDocuments: Exclude<
+        Exclude<ReturnType<typeof useKycAuthStatus>['kyc_auth_status'], undefined>['identity']['supported_documents'],
+        undefined
+    >['idv'];
```

Comment:
i see this type is repeated in packages/account-v2/src/containers/POIFlowContainer/POIFlowContainer.tsx . can we have common type for this